addon/components/leaflet-map.js

```diff
@@ -38,7 +39,7 @@ export default BaseLayer.extend(ContainerMixin, {
   ],
 
   leafletProperties: [
-    'zoom:setZoom', 'center:panTo', 'maxBounds:setMaxBounds', 'bounds:fitBounds'
+    'zoom:setZoom', 'center:panTo:zoomPanOptions', 'maxBounds:setMaxBounds', 'bounds:fitBounds:fitBoundsOptions'
```

Comment:
Why `zoomPanOptions`? Leaflet calls it - http://leafletjs.com/reference.html#map-`panOptions`